src/test/java/org/takes/rs/RsPrintTest.java

```diff
@@ -97,6 +100,33 @@ public void closeBodyEvenWhenExceptionHappens() throws IOException {
         );
     }
 
+    @Test
+    public void simple() throws IOException {
+        final StringWriter writer = new StringWriter();
+        final String firstline = "HTTP/1.1 500 Internal Server Error";
+        final Set<String> head = Collections.singleton(firstline);
+        new RsPrint(new RsSimple(head, "")).printHead(writer);
+        MatcherAssert.assertThat(
+            writer.getBuffer().toString(),
+            Matchers.equalTo("HTTP/1.1 500 Internal Server Error\r\n\r\n")
+        );
+    }
+
+    /**
+     * RFC 7230 says we shall support dashes in response first line.
+     */
+    @Test
+    public void simpleWithDash() throws IOException {
+        final StringWriter writer = new StringWriter();
+        final String firstline = "HTTP/1.1 203 Non-Authoritative";
+        final Set<String> head = Collections.singleton(firstline);
```

Comment:
@rusefillc Like above, let's use `SetOf` here instead of `Collections.singleton()` and inline lines 121 and 122 in line 123